frappe/public/js/frappe/views/communication.js

```diff
@@ -499,7 +501,7 @@ frappe.views.CommunicationComposer = Class.extend({
 		if (this.dialog) {
 			try {
 				let message = this.dialog.get_value('content');
-				message = message.split('<span data-comment="original-reply" class="hidden">Reply To</span>')[0];
```

Comment:
This also needs to be updated in timeline.js

https://github.com/frappe/frappe/blob/d81797c7dde4b409470114f3e6de9761ae771e3e/frappe/public/js/frappe/form/footer/timeline.js#L346-L348

Reply:
fixed